### PR TITLE
chore(doc): Add documentation for using this module

### PR DESCRIPTION
**Pull request by nolanmar511**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,31 +4,123 @@
...
-# Running the system test
-The system test starts a simple benchmark, uses this module to collect a time
-and a heap profile, and then verifies that the profiles can be opened and that
-the profiles contain functions from within the benchmark. 
+## Prerequisites
+1. Your application will need to be using Node.js 8.9.4 or greater, or Node.js 
+10.4.1 or greater. The profiler will not be enabled when using earlier versions
+of Node 8 and 10 because the profiler is not stable with those versions of
+Node.js.
+    * Versions of Node.js before Node.js 8 prior to 8.9.4 are impacted by
```
Comment (kalyanac) [inline README.md]: Versions of Node.js prior to 8.9.4 are affected

Comment (nolanmar511) [reply README.md]: Done.

**Comment by nolanmar511** [comment]
PTAL

File: README.md [hunk 2]
```diff
@@ -4,31 +4,123 @@
...
 
-To run the system test with the v8 canary build, use:
-```console
-$ RUN_ONLY_V8_CANARY_TEST=true sh system-test/system_test.sh
-```
+## Basic Set-up
+
+Install [`pprof`][npm-url] with `npm` or add to your `package.json`.
+
+  ```sh
```
Comment (kalyanac) [inline README.md]: Between the two files, there is mixed usage of 'sh' and 'console', with and without the $ sign. 
Making the shell commands consistent would be nice.

Comment (nolanmar511) [reply README.md]: Done.

**Comment by nolanmar511** [comment]
Ignoring npm audit failure, since it's not related to these changes and only impacts dev dependencies.

**Pull request merged**